HW_7/view.py

```diff
@@ -0,0 +1,14 @@
+def show_menu():
+    question = 'Введите необходимую команду:\
+        \n1 - Показать все контакты\
+        \n2 - Открыть файл с контактами\
+        \n3 - Записать файл с контактами\
+        \n4 - Добавить контакт\
+        \n5 - Поиск по контактам\
+        \n0 - Выход\
+        \n'
+    command = input(question)
+    return command
+
+def show_contacts(contacts: list):
+    [print(contact) for contact in contacts]
```

Comment:
Зачем же так делать, когда лучше и правильней сделать так:
```suggestion
    print(*(contact for contact in contacts), sep='\n')
```